internal/app/scale/woops.go

```diff
@@ -0,0 +1,67 @@
+package scale
```

Comment:
lol what is this

Reply:
haha I forgot about that file.  #doublewoops